rust/ql/lib/codeql/rust/internal/TypeInference.qll

```diff
@@ -2594,6 +2668,72 @@ private module NonMethodResolution {
     ArgsAreInstantiationsOf<NonMethodArgsAreInstantiationsOfInput>;
 }
 
+abstract private class TupleConstructor extends Addressable {
```

Comment:
I think this name is potentially confusing, as its not for tuples but tuple-like structs and variants.
```suggestion
abstract private class TupleLikeConstructor extends Addressable {
```